Applied-Research-Winter-2022/Pozter
# [POZ-10] Setup back end and 1 table for connection testing

## Link Jira ticket:
https://jess-nguy.atlassian.net/browse/POZ-10

## Author Notes:
- This setups node packages: express, joi, dotenv, pg, and nodemon
- I've set up a postgres database with heroku to allow us to use the same data base instead of running migrations and seeds for our local.
- Started with 2 endpoints that work but the table doesn't have all the columns.
billboard: `fetchById`, and `fetchAll`
- Updated Readme to show how to get started and run the backend
- Set up a `.sampleenv` that can be copied into your own `.env`
- Set up a SQL folder to hold table/view creation/updates(alters)

## Impacts and Notes:
NA

## Testing Guidance:
1. Run back end (follow readme)
2. Open webpage or something like insomnia and check if the endpoints work
3. Set url to `http://localhost:8000/billboard/id?id=4e2627c2-9023-11ec-ad70-8b970deef99c`
4. check that it sends some data back. only 1 row
5. `http://localhost:8000/billboards`
6. check that it sends some data back. multiple rows

## Author Checklist:
- [x] Change Jira ticket status to into Review.
- [x] Test have been added or considered. Mentions of considered is noted in PR.
